src/Raven.Studio/typings/_studio/dto.d.ts

```diff
@@ -653,9 +653,15 @@ type timeMagnitude = "minutes" | "hours" | "days";
 
 interface pullReplicationExportFileFormat {
     Database: string;
-    Certificate?: string;
-    HubDefinitionName: string;
+    HubTaskName: string;
```

Comment:
; at the end of lines in interfaces

I wonder if we want to change prop: HubDefinitionName to HubTaskName?

Reply:
as discussed, changing to `HubName`